### ISSUE TITLE
FAKE:100

### ISSUE THREAD

**Issue by lifemo** [Rudrabha/Wav2Lip#426]
Hello,  I have a question for you. When I was training, the loss of Fake became bigger and bigger, and finally it rose to 100, and the loss of percep  value was 0. What is the most likely problem?

**Comment by NikitaKononov** [comment]
The problem is that GAN has failed to train. Most likely problems are: insufficient/bad data, wrong LR and BS

**Comment by lifemo** [comment]
> The problem is that GAN has failed to train. Most likely problems are: insufficient/bad data, wrong LR and BS

Thank you！In addition, I have some questions about when to stop training. "stop whenever eval loss is consistently areater than train loss for ~10 epochs". What is the eval loss？Mine is stable at around 0.08

**Comment by NikitaKononov** [comment]
> > The problem is that GAN has failed to train. Most likely problems are: insufficient/bad data, wrong LR and BS
> 
> Thank you！In addition, I have some questions about when to stop training. "stop whenever eval loss is consistently areater than train loss for ~10 epochs". What is the eval loss？Mine is stable at around 0.08

Validation loss got by eval_model function
0.08? It's reconnection (L1) loss?

**Comment by milely** [comment]
Hello, have you solved this problem? I also encountered this problem. Did you finally find out what caused it?

**Comment by DogeFlow** [comment]
Had the same problem, did you finally solve it

**Comment by dizhenx** [comment]
> Had the same problem, did you finally solve it

Had the same problem, did you finally solve it
